README.md

```diff
@@ -19,3 +19,13 @@ variable to the root of your rls checkout:
 ```
 export RLS_ROOT=/Source/rls
 ```  
+## Manually add as regular VSCode extension
+
+If you'd like to test on multiple projects and already have the extension working properly, want to load the extension by default.
+
+After following the above instructions, and successfully building the extension once, copy the entire `rls_vscode` directory to either:
+```
+Windows: %USERPROFILE%\.vscode\extensions
+Mac/Linux: $HOME/.vscode/extensions
+```
+You can also add `export RLS_ROOT=/Source/rls` to your bash profile or equivalent. More information available via [VSCode docs](https://code.visualstudio.com/Docs/extensions/example-hello-world#_installing-your-extension-locally).
```

Comment:
This note on `RLS_ROOT` feels like it belongs to the above section rather than this one.